src/crud.js

```diff
@@ -1,31 +1,65 @@
-import './style.css';
+const listTasks = JSON.parse(localStorage.getItem('tasks')) || [];
+const listCard = document.getElementById('list');
+const insertInput = document.getElementById('insert');
+const enterBtn = document.getElementById('enterBtn');
 
-const listTasks = [
-  {
-    description: 'Going to Office',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'Meeting with ali',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Go to market',
+const updateIndexes = () => {
+  listTasks.forEach((task, index) => {
+    task.index = index;
+  });
+};
+
+const updateLocalStorage = () => {
+  localStorage.setItem('tasks', JSON.stringify(listTasks));
+};
+
+const deleteTask = (index) => {
+  listTasks.splice(index, 1);
+  updateIndexes();
+  updateLocalStorage();
+};
+
+const renderTasks = () => {
+  listCard.innerHTML = '';
+  listTasks.forEach((task, index) => {
+    const taskElement = document.createElement('div');
+    taskElement.innerHTML = `
+      <div class="card-task">
+        <input type="checkbox" ${task.completed ? 'checked' : ''}>
+        <div class="card list">${task.description}</div>
+        <button class="btn btn-delete" type="button"></button>
+      </div>`;
+    const deleteButton = taskElement.querySelector('.btn-delete');
+    deleteButton.addEventListener('click', () => {
+      deleteTask(index);
+    });
+    taskElement.querySelector('input').addEventListener('change', () => {
+      task.completed = !task.completed;
+      updateLocalStorage();
+      renderTasks();
+    });
+    listCard.appendChild(taskElement);
+  });
+};
+
+const addTask = (description) => {
+  const newTask = {
+    description,
     completed: false,
-    index: 2,
-  },
+    index: listTasks.length,
+  };
```

Comment:
- [ ] By default your index should start at `1` not `0` which is what you currently have in your project. I kindly suggest you add `1` to the index whenever a new task is added this way your index starts at `1`. 👍

Reply:
- [ ] Your project currently doesn't have the functionality for editing a task please kindly implement this as It is a [project requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=Implement%20a%20function%20for%20editing%20task%20descriptions) 
![ed](https://github.com/MohammadYaser/To-do-List/assets/89055502/669b1b5c-d1a8-4439-8269-30979ff01d5a)